### ISSUE TITLE
Warning: DOMDocument::loadHTML(): htmlParseEntityRef: expecting ';' in Entity,

### ISSUE THREAD

**Issue by jcroucher** [PHPOffice/PhpSpreadsheet#647]
To prevent third-party libs which use PhpSpreadsheet from failing on warnings, I think there needs to be a modification to suppress parse warnings either permanently or via a flag

_phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Html.php:546_

$dom = new DOMDocument();
        //    Reload the HTML file into the DOM object
        **$internalErrors = libxml_use_internal_errors(true);**
        $loaded = $dom->loadHTML(mb_convert_encoding($this->securityScanFile($pFilename), 'HTML-ENTITIES', 'UTF-8'));
        **libxml_use_internal_errors($internalErrors);**
        if ($loaded === false) {
            throw new Exception('Failed to load ' . $pFilename . ' as a DOM Document');
        }

https://stackoverflow.com/questions/1685277/warning-domdocumentloadhtml-htmlparseentityref-expecting-in-entity

**Comment by martinresko** [comment]
Why nothing was change? There is still problem with HTMLloading its failing because of it.

**Comment by maisen20** [comment]
Yes, that's still a problem. I'm wondering why not more people have this issue!?

**Comment by sciracioglu** [comment]
when i upgrade laravel 5.8 to 6.0, i had got same error

**Comment by oleibman** [comment]
Reopening. Expect a fix soon.
